all-the-icons.el

```diff
@@ -726,7 +726,8 @@ for performance sake.")
     (lilypond-mode                      all-the-icons-faicon   "music"            :face all-the-icons-green)
     (magik-session-mode                 all-the-icons-alltheicon "terminal"       :face all-the-icons-blue)
     (magik-cb-mode                      all-the-icons-faicon "book"               :face all-the-icons-blue)
-    (meson-mode                         all-the-icons-fileicon "meson"            :face all-the-icons-purple)))
+    (meson-mode                         all-the-icons-fileicon "meson"            :face all-the-icons-purple)
+    (Man-mode                           all-the-icons-fileicon "man-page"         :face all-the-icons-blue)))
```

Comment:
I believe you should specifiy `man-common` instead. Most people with `woman` instead of the vanilla `Man-mode`.

Reply:
Good point, I've changed the PR description, too.